kdl_parser/package.xml

```diff
@@ -22,16 +22,24 @@
 
   <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
 
+  <build_depend>cmake_modules</build_depend>
+  <build_depend>liburdfdom-headers-dev</build_depend>
   <build_depend version_gte="1.3.0">orocos_kdl</build_depend>
   <build_depend>rosconsole</build_depend>
-  <build_depend>roscpp</build_depend>
+  <build_depend>tinyxml</build_depend>
   <build_depend>urdf</build_depend>
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>rostest</build_depend>
 
-  <run_depend version_gte="1.3.0">orocos_kdl</run_depend>
-  <run_depend>rosconsole</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>urdf</run_depend>
+  <build_export_depend>liburdfdom-headers-dev</build_export_depend>
+  <build_export_depend>orocos_kdl</build_export_depend>
+  <build_export_depend>tinyxml</build_export_depend>
+  <build_export_depend>urdf</build_export_depend>
```

Comment:
If these dependencies are exported, they should be exported in CMake as well by using `CATKIN_DEPENDS` or `DEPENDS` right ?

Reply:
Looks like orocos_kdl, tinyxml, and urdf are already being exported in the CMakeLists.txt